pkg/image/controller/signature/container_image_downloader.go

```diff
@@ -62,9 +62,6 @@ func (s *containerImageSignatureDownloader) DownloadImageSignatures(image *image
 		// signature.
 		sig.Name = imageapi.JoinImageStreamImage(image.Name, fmt.Sprintf("%x", sha256.Sum256(blob)))
 		sig.Content = blob
-		sig.Annotations = map[string]string{
-			SignatureManagedAnnotation: "true",
```

Comment:
@mfojtik does the controller do something with this annotation later?

Reply:
No, there's no other reference to the annotation apart those referenced here.

Reply:
no and I don't remember adding this